app/jobs/send_welcome_email.rb

```diff
@@ -1,5 +1,6 @@
 class SendWelcomeEmail < Struct.new(:person_id, :community_id)
 
+  include SessionContextSerializer
```

Comment:
Another question about this. Will deserializing a job work even if the job was originally serialized without session context when the code doing the deserialization has it?